start/sw/web-app/laravel5.md

```diff
@@ -58,10 +58,10 @@ $ composer -V
 $ composer global require "laravel/installer"
 # Place the ~/.config/composer/vendor/bin directory in your PATH so the laravel 
 # executable can be located by your system.
-$ echo 'export PATH="$PATH:$HOME/.config/composer/vendor/bin"' >> ~/.bash_profile
```

Comment:
You do not need the `export` here. PATH should be already exported on all UNIX systems.

For immediate change in current terminal you could run the command `source !$` instead, which sources ~/.bash_profile.